### PR TITLE
Added Ushort and Sbyte type

### PR DESCRIPTION
**Pull request by smaktacular**
Added Types for ushort and sbyte.

### CODE REVIEW

File: src/HotChocolate/Core/src/Types.Scalars/SignedByteType.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
+using HotChocolate.Language;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The UnsignedInt scalar type represents a unsigned numeric non‐fractional
```
Comment (michaelstaib) [inline src/HotChocolate/Core/src/Types.Scalars/SignedByteType.cs:6]: this comment still seems to be off ... UnsignedInt but we are a SignedByte

Comment (smaktacular) [reply src/HotChocolate/Core/src/Types.Scalars/SignedByteType.cs]: yeah ... reading things carefully is a valuable trait ...
done

**Review by michaelstaib — APPROVED** [review]

**Comment by michaelstaib** [comment]
This one is now read for merge. Can you do a second PR on this documentation site and add your scalar to the table of scalars?

Web:
https://chillicream.com/docs/hotchocolate/defining-a-schema/scalars/#additional-scalars

The page can be found here:
https://github.com/ChilliCream/hotchocolate/blob/main/website/src/docs/hotchocolate/defining-a-schema/scalars.md

**Pull request merged**